projects/1.16/assets/hats/hats/lang/zh_cn.json

```
@@ -1 +1,244 @@
+
+  "hats.toast.sync.receivedHat": "从服务器收到了帽子",
+
+  "hats.gui.selection.title": "帽子菜单",
+
+  "hats.gui.window.hatsList": "帽子列表",
+  "hats.gui.window.sidebar": "侧边栏",
+
+  "hats.gui.window.hat.sidebar": "帽子侧边栏",
+  "hats.gui.window.hat.colorizer": "帽子着色器",
+  "hats.gui.window.hat.personaliser": "帽子配饰器",
+  "hats.gui.window.hat.noHat": "没有帽子？也许应该检查一下你的过滤器。",
+
+  "hats.gui.window.hatsList.title": "你的帽子",
+  "hats.gui.window.allHats.title": "所有帽子变种",
+  "hats.gui.window.allHats.hatsOutOf": "%1$s/%2$s个",
+
+  "hats.gui.window.sorter.title": "帽子过滤器和排序器",
+  "hats.gui.window.sorter.filters": "过滤器",
+  "hats.gui.window.sorter.sorters": "排序器",
+  "hats.gui.window.sorter.enabled": "启用",
+  "hats.gui.window.sorter.enabledInverse": "启用-反向",
+  "hats.gui.window.sorter.disabled": "禁用",
+  "hats.gui.window.sorter.dragTip": "你可以拖动排序器",
+
+  "hats.gui.window.management.title": "帽子管理",
+  "hats.gui.window.management.reloadAllHats": "重新加载所有帽子",
+  "hats.gui.window.management.reloadedAllHats": "已重新加载所有帽子。",
+  "hats.gui.window.management.synchroniseWithServer": "与服务器同步",
+  "hats.gui.window.management.sync.waiting": "等待服务器回应……",
+  "hats.gui.window.management.sync.noDifferences": "一切顺利！帽子名称在两边都匹配",
+  "hats.gui.window.management.sync.issue": "同步问题！",
+  "hats.gui.window.management.sync.clientMissing": "客户端缺少：",
+  "hats.gui.window.management.sync.serverMissing": "服务器缺少：",
+  "hats.gui.window.management.sync.syncReq": "要不要和服务器同步？",
+
+  "hats.gui.tutorial.title": "教程",
+  "hats.gui.tutorial.toast.title": "要打开你的帽子列表",
+  "hats.gui.tutorial.toast.subtitle": "按下“%s”键",
+  "hats.gui.tutorial.intro": "你想开始GUI教程吗？",
+  "hats.gui.tutorial.noHat": "哎呀，看起来你还没有帽子。\n\n我们已经向服务器要了一顶给你。\n\n别担心，那个算我请客！",
+  "hats.gui.tutorial.camera": "你可以用鼠标调整左边的画面。\n\n随便用鼠标按键和滚轮点击和拖动。\n\n现在就去试试吧！",
+  "hats.gui.tutorial.cameraFallback": "你可以用鼠标来控制左边的视角。\n\n通常，你可以用鼠标的按键和滚轮来点击和拖动，但是我们现在处于后备模式，所以也许等一下再试吧？",
+  "hats.gui.tutorial.hatsHere": "你解锁的帽子在这里。\n\n（不要现在点击，教程的互动部分已经结束了）",
+  "hats.gui.tutorial.clickForHatMissing": "你现在应该有一顶帽子，但是我们好像找不到。\n\n也许你的过滤器把它们隐藏了？\n\n算了……",
+  "hats.gui.tutorial.clickForHat": "你可以点击一顶帽子来戴上它，或者右键点击获得更多选项。",
+  "hats.gui.tutorial.search": "你可以在此处搜索特定的帽子。\n\n你也能右键点击来清除输入",
+  "hats.gui.tutorial.additionalButtons": "上面有一些额外的按钮。\n\n鼠标移到上面可以看到提示。",
+  "hats.gui.tutorial.dragBar": "需要更多空间来看你的帽子吗？\n\n把这个拖到左边。",
+  "hats.gui.tutorial.conclusion": "就这样了！\n\n谢谢你的游玩，享受帽子吧！",
+  "hats.gui.tutorial.restart": "重新开始GUI教程",
+  "hats.gui.tutorial.skip": "当然。你可以随时在帽子资源管理里重新开始教程",
+
+  "hats.gui.button.removeHat": "移除帽子",
+  "hats.gui.button.randomHat": "随机帽子\n +SHIFT: 随机色调",
+  "hats.gui.button.randomHatLauncher": "随机帽子",
+  "hats.gui.button.sortingOptions": "排序选项",
+  "hats.gui.button.hatResourceManagement": "帽子资源管理",
+
+  "hats.gui.button.colourise": "着色",
+  "hats.gui.button.personalise": "配饰",
+  "hats.gui.button.favouriteHat": "收藏",
+
+  "hats.gui.button.rgb": "RGB调节器",
+  "hats.gui.button.hsb": "HSB调节器",
+  "hats.gui.button.glint": "发光效果",
```

```suggestion
  "hats.gui.button.glint": "光效",
```
不知道是不是这个
https://minecraft.fandom.com/zh/wiki/%E9%99%84%E9%AD%94